components/NavBar/NavBar.tsx

```diff
@@ -197,12 +201,11 @@ const NavBar = (): ReactElement => {
 						urls={[INTERNAL_PATHS.ACKNOWLEDGEMENTS, INTERNAL_PATHS.TEAM]}
 					/>
 
-					<InternalLink path={INTERNAL_PATHS.RELEASES}>
-						<StyledLink
-							className={cx({ active: router.asPath.startsWith(INTERNAL_PATHS.RELEASES) })}
-						>
-							Data Releases
-						</StyledLink>
+					<InternalLink
+						active={router.asPath.startsWith(INTERNAL_PATHS.RELEASES)}
+						path={INTERNAL_PATHS.RELEASES}
+					>
+						<StyledNavBarLink>Data Releases</StyledNavBarLink>
```

Comment:
can you share some insight on the motivation behind this change in pattern? it's not immediately clear here

e.g. passing "active" as a props into nextjs' link only to make it a classname there, instead pf passing the classname directly (which I wasn't aware was an option)... at which point what's the purpose of this nested "StyledLink" anchor?

Reply:
This has since been updated, see threads w/ Ciaran re: motivation for changes